packages/www/components/Dashboard/WebhookDialog/index.tsx

```diff
@@ -151,6 +155,16 @@ const WebhookDialog = ({
               placeholder="https://"
               onChange={(e) => setWebhookUrl(e.target.value)}
             />
+
+            <Label htmlFor="sharedSecret">Secret</Label>
+            <TextField
+              size="2"
+              type="text"
+              id="sharedSecret"
+              defaultValue={Action.Update ? webhook?.sharedSecret : ""}
+              placeholder="secret used to sign the webhook requests"
+              onChange={(e) => setWebhookSecret(e.target.value)}
+            />
```

Comment:
Would be nice if we could "suggest" a random secret on webhook creation, say like generating a UUID here and pre-populating the shared secret field 🤔 
Would make it less easy for users to create some potentially insecure secrets

Reply:
yes 💯 , will do